.github/workflows/release-manual.yaml

```
@@ -0,0 +1,88 @@
+# This workflow only build and upload to CF R2 for a specific contract name passed as input.
+name: Manual Release (Only build and upload to CF R2)
```

```suggestion
name: Upload to R2
```
provide a more concise name